brave/brave-ios
# Fix #3014: Adding Open Tabs in Search Suggestions

Typing text into omni-box (url bar) matches against open tabs and includes suggestions to section under the search keyword suggestions.

This section is renamed as Open Tabs & Bookmarks & History and every entry is marked with a right badge image according to their type.

The Open Tabs entry is also marked as **Switch to this tab** in detailed text before url is shown.

Pressing Open Tab entry should open (existing tab) instead of opening a new one. 

## Summary of Changes


This pull request fixes #3014

## Submitter Checklist:


## Test Plan:
- Search "test" and open one of the suggestions in a tab
- Create a new tab and search for "test"
- Scroll down in search suggestions and find out Open Tab suggestion for the particular open tab
- Click and check existing tab is opened

## Screenshots:


![tabs open](https://user-images.githubusercontent.com/6643505/160012840-ce671e1e-5eab-4034-85a1-a4340ea21b88.png)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
